management/ssl_certificates.py

```diff
@@ -249,6 +250,7 @@ def rdata__str__(r):
 				s = r.to_text()
 				if isinstance(s, bytes):
 					 s = s.decode('utf-8')
+				s = str(ipaddress.ip_address(s))
```

Comment:
Well, not all instances throughout the project. But if you replaced just the cases where it is necessary for fixing the original issue, yes.

But better would be to create a function --- probably in status_checks.py --- called `normalize_ip` address, where you put the `str(ipaddress...)` logic, and then call `normalize_ip` wherever you would want to have used `ipaddress.ip_address`, so that the normalization logic is in one place.